### DISCUSSION TITLE
Cannot install

### DISCUSSION THREAD

**Discussion by xiangtp** [windingwind/zotero-better-notes#689] [Q&A] [answered]
Hi everyone,

I try to install Better Notes add-on on my Zotero, using the lates .xpi file, but it says "cannot be installed". The reason predicted was not compatible my Zotero version. I am using 6.0.27 version. Below is the screenshot.

Please advise. Thank you!

![image](https://github.com/windingwind/zotero-better-notes/assets/144694440/f8bbef99-c98a-4459-8876-ca0a3f18f5b5)

**Comment by windingwind** [comment, marked as answer]
see readme#install

**Reply by xiangtp** [reply]
Thanks. I have read and followed that, but it does not work. Do you have any advice regarding the version?

**Reply by paulgronke** [reply]
I am also encountering the same error. The plug in will not install on Zotero 6.0.27, running  Mac OS
